libraries/xcode.rb

```diff
@@ -2,10 +2,24 @@ module Xcode
   module Helper
     BASE_COMMAND ||= '/usr/local/bin/xcversion'.freeze
 
-    def xcode_already_installed?(version)
-      xcversion_output = shell_out!("#{BASE_COMMAND} installed").stdout.split
+    def xcode_already_installed?(semantic_version)
+      xcversion_output = shell_out("#{BASE_COMMAND} installed").stdout.split
       installed_xcodes = xcversion_output.values_at(*xcversion_output.each_index.select(&:even?))
-      installed_xcodes.include?(version)
+      installed_xcodes.include?(semantic_version)
+    end
+
+    def xcversion_version(version)
+      semantic_version = version.split('.')
+      if semantic_version.length == 2 && semantic_version.last == '0'
+        semantic_version.first
+      else
+        version
+      end
```

Comment:
Let's split this up into two different methods to make it a bit more readable:

```ruby
def needs_shortening?(version)
  split_version = version.split('.')
  split_version.length == 2 && split_version.last == '0'
end

def xcversion_version(full_version)
  full_version.first ? needs_shortening?(full_version) : full_version
end
```

Reply:
```
[2] pry(main)> full_version.first
NoMethodError: undefined method `first' for "9.0":String
```

I like the boolean method idea but I'd need to add another `split('.')` for `full_version` in the `xcversion_version` for correct output, which seems less elegant. I've updated the variable names for better clarity.